cmd/finch/nerdctl.go

```diff
@@ -98,6 +263,22 @@ func (nc *nerdctlCommand) runAdapter(cmd *cobra.Command, args []string) error {
 	return nc.run(cmd.Name(), args)
 }
 
+func (nc *nerdctlCommand) runDockerCompatInspectAdapter(cmd *cobra.Command, args []string) error {
```

Comment:
All of these functions are only called once each. Are they here just for easier unit testing?